### PR TITLE
Minor rewording in the backgrounds page

### PR DESCRIPTION
**Pull request by bsipocz**
@dpshelio - I've changed a few sentences, please have a look whether you like it or not.

### CODE REVIEW

File: gsoc/background.md [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
-previous experience in Python and/or astronomy. Underrepresented students from
-anywhere around the world are especially encouraged to apply.
+OpenAstronomy is actively seeking applicants who are undergraduate or
+graduate students with experience developing software for astronomy in
+Python and other languages. We are especially (but not exclusively) looking
+for students who have or are pursuing degrees in astronomy or physics, more
+importantly who are interested in developing open source software. You don't
+need to be a Python guru or an astronomical coordinate system expert to be
+eligible -- projects are available that cover a range of previous experience
+in Python and astronomy. Underrepresented students from anywhere around
```
Comment (bsipocz) [inline gsoc/background.md]: Is the word `underrepresented` necessary? What does it even mean in the around the world content? The bottom line is really that anyone is really welcome to apply and has a real chance to get selected.

Comment (mirca) [reply gsoc/background.md]: I would prefer to emphasize something like ``Any student from any background is very welcome to apply`` instead.

Comment (bsipocz) [reply gsoc/background.md]: Yes, I would prefer that one, too. What do you think @dpshelio?

Comment (bsipocz) [reply gsoc/background.md]: "Students from any background from anywhere around the world is very welcome to apply." ?

Comment (dpshelio) [reply gsoc/background.md]: I agree with @mirca :)

**Comment by bsipocz** [comment]
Merging this now. For more fine tuning, please open a follow-up PR.

**Pull request merged**